routemaster/config/visualisation.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<style>
+body {
+  font: 14px helvetica neue, helvetica, arial, sans-serif;
+  margin: 0;
+}
+#cy {
+  height: 100%;
+  width: 100%;
+  position: absolute;
+  left: 0;
+  top: 3.5em;
+}
+#controls {
+  background-color: #f0f0f0;
+  margin: 0;
+}
+#controls div {
+  display: inline-block;
+  padding: 0.5em;
+  margin-right: 0.2em;
+}
+#spacing-factor-input-container {
+  top: 0.5em;
+  position: relative;
+}
+.muted {
+  color: #666666;
+}
+.action {
+  background-color: orange;
+}
+.gate {
+  background-color: lightBlue;
+}
+</style>
+<title>Visualisation of state machine {{ state_machine_name }}</title>
+</head>
+<body>
+  <script type="text/javascript">
+    var state_machine_config = {{ state_machine_config|tojson }};
+  </script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/cytoscape/3.2.11/cytoscape.js"></script>
+  <div id="controls">
+    <div id="legend">
+      <div class="action">action</div>
+      <div class="gate">gate</div>
+    </div>
+    <div>
+      <label>
+        Spacing:
+        <span id="spacing-factor-input-container">
+          <input id="spacing-factor" type="range" onchange="changeSpacing(this.value / 10); return false;" min="1" max="20" step="0.1">
+        </span>
+        <span id="spacing-factor-display"></span>
+      </label>
+    </div>
+    <div class="muted">Click and drag to move the graph</div>
+  </div>
+  <div id="cy"></div>
```

Comment:
"graph" might be a more obvious name, clearer that this is the main component.

Reply:
`#cy` is the name in the example for the library. I agree that we can do better: 11a9e81.